package.json

```diff
@@ -113,6 +113,7 @@
     "stylelint-config-standard": "34.0.0",
     "stylelint-order": "6.0.3",
     "stylelint-selector-bem-pattern": "3.0.1",
+    "tailwindcss": "^3.3.3",
```

Comment:
Should tailwind be a devDependency? I thinbk it should be a production dependency, not?